mifosng-android/src/main/java/com/mifos/objects/client/DocumentRelatedResponse.java

```diff
@@ -0,0 +1,64 @@
+package com.mifos.objects.client;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by Tarun on 14-08-17.
+ */
+
+public class DocumentRelatedResponse implements Parcelable {
```

Comment:
Why does this class implement `Parcelable` ?

Reply:
I didn't use the parcelable attribute but since we have been following this approach to make the classes parcelable (probably to be used later; e.g. while saving the instance state), I did so. Should I remove it?

Reply:
Yes please :) Thanks

Reply:
Done! :)